assets/styles/jon.scss

```diff
@@ -0,0 +1,8 @@
+#test{
+  .name1{
```

Comment:
Selector should have depth of applicability no greater than 1, but was 2<br>Opening curly brace `{` should be preceded by one space